l10n_it_delivery_note/models/stock_picking.py

```diff
@@ -436,6 +436,8 @@ def write(self, vals):
     def _create_backorder(self):
         """When we make a backorder of a picking the delivery note lines needed
         to be updated otherwise stock_delivery_note_line_move_uniq constraint is raised"""
+
```

Comment:
questa riga vuota serve?

Reply:
mi falliva il precommit in locale provo a ritoglierla asp

Reply:
sistemato ora gli è piaciuta